launch/launch/actions/execute_process.py

```diff
@@ -324,6 +324,12 @@ def parse(
         :param: ignore A list of arguments that should be ignored while parsing.
             Intended for code reuse in derived classes (e.g.: launch_ros.actions.Node).
         """
+        if ignore is None:
+            entity.assert_attribute_names((
+                'cmd', 'cwd', 'name', 'launch-prefix', 'output', 'respawn', 'respawn_delay', 'shell',
+                'additional_env', 'if', 'unless'
```

Comment:
@ivanpauno hmm, do we have to explicitly list base classes' attributes? What's going to happen with the assertions performed by another class that derives from this one?

Reply:
That's why I used the trick `if ignore is None:`.
Not a beauty, but I don't know how to do better.